src/index.html

```diff
@@ -40,163 +40,177 @@
+                            <span class="nav-caret"></span>
+                        </li>
+                    </ul>
+                </div>
             </div>
         </div>
-    </div>
-    <div id='close_banner_container' class='container invisible'>
-        <div class='close_banner_text'>
-            <img class='close_banner_img' src="v<version>/images/banner/icon_left.png"/>
-            <div class='close_banner_text_wrapper'>
-                <h3>Binary.com is moving to Deriv</h3>
-                <p>Start using Deriv with your Binary.com email and password.</p>
+        <div id="header">
+            <div id="container">
+                <div class="logo-parent">
+                    <a id="logo">
+                        <div class="logo">
+                            <div></div>
+                        </div>
+                        <div class="binary-logo-text">
+                            <div></div>
+                        </div>
+                    </a>
+                </div>
             </div>
         </div>
-        <div class='close_banner_btn'>
-            <a id='close_banner_btn_iom' target="_blank" href="#">Trade on Deriv</a>
-        </div>
-    </div>
-    <div id="content">
-        <div class="container">
-            <div class="text">
-                <h1>Welcome to WebTrader</h1>
-                <p class="paragraph">
-                    WebTrader is the platform that lets you trade and view multiple markets on one screen. Add dashboards and charts, and drag and drop them to create your own layout. Copy trades from successful traders and mirror their activity. 
-                </p>
-                <b>Customisable workspaces</b>
-                <p class="paragraph">
-                    Create multiple custom screens and save each one as a workspace that you can view instantly later. 
-                </p>
-                <b>Trade multiple markets</b>
-                <p class="paragraph">
-                    Trade a multitude of assets in forex, commodities, stock indices, and synthetic indices. 
-                </p>
-                <b>Add useful tools</b>
-                <p class="paragraph">
-                    Use tools like historical data and the asset index on the same screen to inform your trading strategy. 
-                </p>
-                <b>Copy trading</b>
-                <p class="paragraph--br">
-                    Replicate the trades of more experienced traders in your own account.
-                </p>
-               <button onclick="openTradingPage()">Start trading</button>
+        <div id='close_banner_container' class='container invisible'>
+            <div class='close_banner_text'>
+                <img class='close_banner_img' src="v<version>/images/banner/icon_left.png"/>
+                <div class='close_banner_text_wrapper'>
+                    <h3>Binary.com is moving to Deriv on 30 November</h3>
```

Comment:
30 November is already passed, do we still need to include it?

Reply:
also isn't it `30th` 😄

Reply:
actually there is no need for the banner since we redirect all of those clients to 'move-to-deriv', but as @prince said it should be just left like that